spec/fhir/mongoid/measure_spec.rb

```diff
@@ -57,7 +57,88 @@
     # puts cqm_measure.to_json
   end
 
-  it 'should serialize and deserialize' do
+  it 'Should permit updates to CQM Measure from JSON' do
+    cqm_measure = load_measure
+    source_data_criteria = CQM::DataElement.new
+    source_data_criteria.fhir_resource = FHIR::Condition.transform_json @fhir_resources_hash['condition']
+    cqm_measure.source_data_criteria << source_data_criteria
+
+    expect do
+      cqm_measure.save!
+    end.to_not raise_error
+
+    updated_cqm_measure_hash = cqm_measure.as_json
+    updated_cqm_measure_hash['source_data_criteria'][0]['description'] = 'new title'
+    updated_cqm_measure = CQM::Measure.transform_json(updated_cqm_measure_hash)
+
+    expect do
+      updated_cqm_measure.upsert
```

Comment:
Just wanted to verify there that we can update a CQM measure. All nested FHIR resources will lose ids when marshaled to JSON and unmarshaled from JSON. Only the root id is preserved.